SCClassLibrary/Common/Core/AbstractFunction.sc

```diff
@@ -358,3 +358,73 @@ FunctionList : AbstractFunction {
 	storeArgs { ^[array] }
 	copy { ^super.copy.array_(array.copy) }
 }
+
+
+
+Operand : AbstractFunction {
+	var <>value;
+
+	*new { |value|
+		^super.newCopyArgs(value.dereferenceOperand)
+	}
+
+	dereferenceOperand {
```

Comment:
Why not use `dereference`? Or is storing a `Ref` in an operand a valid use case?